attack-theme/templates/general/base-template.html

```diff
@@ -72,23 +73,25 @@
                 <p>
                      © 2015-2020, The MITRE Corporation. MITRE ATT&CK and ATT&CK are registered trademarks of The MITRE Corporation.
                 </p>
-                <div class="row">
-                    <div class="col text-right">
-                        <small>
-                            <a href="/resources/privacy" class="footer-link">Privacy Policy</a>
-                        </small>
-                    </div>
-                    <div class="col text-center">
-                        <small>
-                            <a href="/resources/terms-of-use" class="footer-link">Terms of Use</a>
-                        </small>
-                    </div>
-                    <div class="col text-left">
-                        <small>
-                            <a href="{{CHANGELOG_LOCATION}}" class="footer-link" data-toggle="tooltip" data-placement="top" title="ATT&amp;CK content version {{CONTENT_VERSION}}&#013;Website version {{WEBSITE_VERSION}}">ATT&CK v{{CONTENT_VERSION}}</a>
-                        </small>
+                {% if RESOURCES %}
+                    <div class="row">
+                        <div class="col text-right">
+                            <small>
+                                <a href="/resources/privacy" class="footer-link">Privacy Policy</a>
+                            </small>
+                        </div>
+                        <div class="col text-center">
+                            <small>
+                                <a href="/resources/terms-of-use" class="footer-link">Terms of Use</a>
+                            </small>
+                        </div>
+                        <div class="col text-left">
+                            <small>
+                                <a href="{{CHANGELOG_LOCATION}}" class="footer-link" data-toggle="tooltip" data-placement="top" title="ATT&amp;CK content version {{CONTENT_VERSION}}&#013;Website version {{WEBSITE_VERSION}}">ATT&CK v{{CONTENT_VERSION}}</a>
+                            </small>
```

Comment:
I think we still need the terms of use and changelog in the footer even when the branding is removed. I realize those are resource pages, but they are important outside of the context of resources, especially for the terms of use which explicitly states that it must be present in any reproduction. 

I don't think the privacy policy is relevant since it applies to the live site, so that removal is fine.

Reply:
I know the terms of use also links to the FAQ, another resource page. We can just set the FAQ link to point to https://attack.mitre.org/resources/faq/ explicitly like we already do in the changelog for hyperlinks.

Reply:
We could move the license and changelog out of the resources folder, provided we add redirects. IMO they aren't _really_ resources in the way the other resource pages are.

Reply:
We could add those to the `website_build` module, that way they are always there

Reply:
Yes that would be fine.